client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import CardItem from '../CardItem';
+import styles from './CardContainer.module.scss';
+
+
+import 'swiper/swiper.min.css'
+
+
+
+
+const cards = [
+    {
+        name: "Kyiv city walk",
+        region: "Kyiv region",
+        id: 1,
+    },
+    {
+        name: "Lviv city walk",
+        region: "Lviv region",
+        id: 2,
+    },
+    {
+        name: "Donetsk city walk",
+        region: "Donetsk region",
+        id: 3,
+    },
+    {
+        name: "Odessa city walk",
+        region: "Odessa region",
+        id: 4,
+    },
+    {
+        name: "Dnepr city walk",
+        region: "Dnepr region",
+        id: 5,
+    },
+    {
+        name: "Chernivtsi city walk",
+        region: "Chernivtsi region",
+        id: 6,
+    },
+    {
+        name: "Ternopil city walk",
+        region: "Ternopil region",
+        id: 7,
+    },
+    {
+        name: "Lutsk city walk",
+        region: "Lutsk region",
+        id: 8,
+    },
+]
+
+
+const CardContainer = () =>
+
+    <div className={styles.cardContainer}>
```

Comment:
You have to rewrite this with MUI tags. For example, div will be a <Box><Box/>. Styles can be mentioned as sx props (<Box sx={{padding: "20px"}}>Some content<Box/> or mentioned as a component before CardContainer component (if you have more than 2 styles to add).

Tags here be the following:
div - <Box>Content<Box/>
h2 - <Typography variant="h2">Content<Typography/>

Also, you don`t need to use classnames here. We use MUI and it`s better to follow it`s syntax.
To style your components you can do the following:
1) If you use classname on a component, create a box component before CardContainer in the following way:
const CardWrapper= styled('div')({
  maxWidth: 1150px,
  margin: 0 auto,
  padding: 0 20px,
});
(styles are just copied from your SCSS file and syntax changed for CammelCase)
2) Then, call this component in your code instead of div.